src/Backend/Modules/Pages/Layout/Templates/Edit.html.twig

```diff
@@ -176,7 +176,9 @@
                 <div class="content-block">
                   <div class="content-title">
                     <p>{{ 'lbl.PreviousVersions'|trans|ucfirst }}
-                      <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpRevisions'|trans }}"></abbr>
+                      <a href="#" class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpRevisions'|trans }}">
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320